CHANGELOG.md

```diff
@@ -17,10 +17,10 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ### Bug fixes & Enhancements:
 - [#312] (https://github.com/HewlettPackard/terraform-provider-oneview/issues/312) Unable to apply server_profile_template (Invalid character '.)
```

Comment:
please fix these lines for Bug, those spaces between [#123] and (url) is not required making formatting incorrect